roles/pre_ohpc/templates/hosts.j2

```diff
@@ -2,5 +2,5 @@
 ::1 localhost
 
 {% for host in groups['headnode'] %}
-{{ headnode_private_ip }} {{ hostvars[host]['inventory_hostname'] }}  {{ hostvars[host]['inventory_hostname'] }}.local
+{{ hostvars[inventory_hostname]['ansible_eth1']['ipv4']['address'] }} {{ hostvars[host]['inventory_hostname'] }}  {{ hostvars[host]['inventory_hostname'] }}.local
```

Comment:
Why choose ansible_eth1 and not ansible_eth0?

Reply:
Slurm communication from OHPC to the compute nodes happen through the private network in our setup so I think it should be `ansible_eth1`.

Reply:
I agree that the private network is a good choice.  Only push back here is that it's possible to have just one nic, so eth1 wouldn't exist in that situation but eth0 would exist in all situations.